docs/diagrams/ShowEvents.puml

```diff
@@ -13,7 +13,7 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("showevents")
+[-> LogicManager : execute("listevents")
```

Comment:
should the label `updateFilteredEventList(PREDICATE_SHOW_ALL_EVENTS)` change since by default we are showing events after the current date now? or alternatively we can change the command in the diagram to execute ("listevents -a")

Reply:
Fixed!